packages/components/src/site-thumbnail/index.tsx

```diff
@@ -1,45 +1,70 @@
+import { css, CSSObject } from '@emotion/css';
 import classnames from 'classnames';
 import { ReactNode } from 'react';
 import './style.scss';
 import { MShotsOptions, useMshotsImg } from './use-mshots-img';
 import { getTextColorFromBackground } from './utils';
 
-const MSHOTS_OPTION: MShotsOptions = {
-	vpw: 1200,
-	vph: 1200,
-	w: 374 * 2,
+type SizeCss = CSSObject & { width: number; height: number; sizes?: string };
+const SIZES: { [ sizeName: string ]: SizeCss } = {
+	small: {
+		width: 108,
+		height: 78,
+	},
+	medium: {
+		width: 401,
+		height: 401 * ( 11 / 16 ),
```

Comment:
from a readability standpoint would it make sense to do something like
``
const ASPECT_RATIO = 11 / 16 (assuming this is the aspect ratio)
...
height: 401 * ASPECT_RATIO

``

Reply:
done!